shared/actions/config/avatar.js

```diff
@@ -58,12 +54,12 @@ function* avatarCallAndHandle(names: Array<string>, method: Function) {
   }
 }
 
-let avatarChannel
 function* handleAvatarQueue() {
-  avatarChannel = yield Saga.channel(Saga.buffers.dropping(1))
   while (true) {
-    yield Saga.callUntyped(Saga.delay, 100)
-    yield Saga.take(avatarChannel)
+    // nothign in queue, keep listening
+    if (!avatarsToLoad.users.size && !avatarsToLoad.teams.size) {
```

Comment:
handle this more like we do in chat now. we dont need our own channel for this kind of thing